src/commands/claim.js

```diff
@@ -10,6 +11,15 @@ exports.run = async function(payload, commenter, args) {
     });
   }
 
+  if (payload.issue.assignees.length >= 1 && !claimed) {
```

Comment:
`payload.issue.assignees.length` should be fine

Reply:
Just a gentle reminder, made changes. Ready for review :)